src/audioWorklet/index.js

```diff
@@ -1,4 +1,4 @@
-import p5sound from '../master.js'
```

Comment:
it looks like this commit includes some lint fixes. Thank you! In the future, I recommend only adding specific files to the commit, to avoid adding files that aren't related to the PR. `git add -p` (https://gist.github.com/mattlewissf/9958704) is really useful for this, it lets us choose which modifications we want to add to staging. That way, we can commit these sorts of auto-style fixes separately.